llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-splat-vector.mir

```diff
@@ -7,6 +7,78 @@
 # (c) Copyright 2023-2024 Advanced Micro Devices, Inc. or its affiliates
 # RUN: llc -mtriple aie2 -run-pass=instruction-select %s -verify-machineinstrs -o - | FileCheck %s
 
+---
```

Comment:
nit: these tests are redundant. `G_AIE_BROADCAST_VECTOR` always produces a 512-bit vector, for which we already have tests